unicorn_mode/helper_scripts/unicorn_dumper_pwndbg.py

```diff
@@ -111,12 +111,14 @@ def dump_regs():
     reg_state = {}
     for reg in pwndbg.gdblib.regs.all:
         reg_val = pwndbg.gdblib.regs[reg]
+        if not reg_val:
```

Comment:
This will mean reg_val of `0` will never be dumped, I don't think this is correct? This should likely be `if reg_val is None`? Or what's the meaning of this?

Reply:
Yes, you are right

Reply:
Do you have time to fix it?

Reply:
> Do you have time to fix it?

I have fixed it.